lib/cc_plugin_imos/cc_plugin_imos/tests/test_imos.py

```diff
@@ -306,7 +300,52 @@ def test_vertical_coordinate_type(self):
 
 
 
-    ### Test compliance checks
+################################################################################
+#
+# IMOS 1.3 Checker
+#
+################################################################################
+
+class TestIMOS1_3(unittest.TestCase):
```

Comment:
Yes, but it would also inherit all the `test_...` methods that we don't need to repeat. I'd have to create a separate base class with just the shared methods, and then `TestUtils` and `TestIMOS1_3` could both inherit from that. I can do it in a future PR.